Index: healthScore/forms.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from django import forms\nfrom .models import Post\n\nclass PostForm(forms.ModelForm):\n    class Meta:\n        model = Post\n        fields = ['title', 'description']\n\n
===================================================================
diff --git a/healthScore/forms.py b/healthScore/forms.py
--- a/healthScore/forms.py	
+++ b/healthScore/forms.py	
@@ -1,8 +1,13 @@
 from django import forms
-from .models import Post
+from .models import Post, Comment
+
 
 class PostForm(forms.ModelForm):
     class Meta:
         model = Post
         fields = ['title', 'description']
 
+class CommentForm(forms.ModelForm):
+    class Meta:
+        model = Comment
+        fields = ['content']
\ No newline at end of file
Index: healthScore/templates/create_post.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><!DOCTYPE html>\n<html lang=\"en\">\n<head>\n    <meta charset=\"UTF-8\">\n    <title>Title</title>\n</head>\n<body>\n{% block content %}\n    <h2>Create Post</h2>\n    <form method=\"post\">\n        {% csrf_token %}\n        <p>\n            <label for=\"id_title\">Title:</label>\n            <input type=\"text\" name=\"title\" id=\"id_title\"/>\n        </p>\n        <p>\n            <label for=\"id_description\">Description:</label>\n            <textarea name=\"description\" id=\"id_description\"></textarea>\n        </p>\n\n        <button type=\"submit\">Post</button>\n    </form>\n{% endblock %}\n</body>\n</html>
===================================================================
diff --git a/healthScore/templates/create_post.html b/healthScore/templates/create_post.html
--- a/healthScore/templates/create_post.html	
+++ b/healthScore/templates/create_post.html	
@@ -17,9 +17,9 @@
             <label for="id_description">Description:</label>
             <textarea name="description" id="id_description"></textarea>
         </p>
-
         <button type="submit">Post</button>
     </form>
+
 {% endblock %}
 </body>
 </html>
\ No newline at end of file
